app/lib/income_section.rb

```diff
@@ -5,7 +5,7 @@ def all_steps
     end
 
     def steps_for(estimate)
-      if estimate.passporting
+      if estimate.passporting || estimate.asylum_support
```

Comment:
To avoid a possible edge case we should add this to the three partner sections as well: If a user initially answers `no` to the asylum support question, then on the applicant details screen indicates they have a partner, then modifies their responses at some point to now say `yes` to the asylum support question, we need to be sure that they are _not_ then prompted to fill out the partner details.